trunk/SUAVE/Components/Component.py

```diff
@@ -22,7 +22,7 @@ class Component(Data):
     def __defaults__(self):
         self.tag    = 'Component'
         self.origin = [[0.0,0.0,0.0]]
-    
+        #self.origin = np.array([0,0,0])

```

Comment:
Emilio made a change to the code, such that the default component was a list of lists, rather than a numpy array (this was to accommodate things like quadcopters, where each rotor has a different origin) . I put it in to see what it does, and figure out where the changes were, and what they would be. I can take it out if you want